src/vec_avx.h

```diff
@@ -168,8 +168,8 @@ typedef struct {
   __m128i lo;
   __m128i hi;
 } mm256i_emu;
-typedef __m256i real_m256i;
 #define __m256i mm256i_emu
+typedef __m256i real_m256i;
```

Comment:
Doesn't this simply become `typedef mm256i_emu real_m256i` after preprocessing?

Reply:
Yes, but `mm256i_emu` is also used in other places.

Reply:
But if both are always the same type, there wouldn't be any need to define [this union](https://github.com/xiph/rnnoise/blob/27a25bb4cc3ba27ccbe0c844335944af8434d4dd/src/vec_avx.h#L254-L257).

Note: I'm not familiar with the codebase, just started exploring for a project. Just pointing out a logical inconsistency.